src/gc/gc.d

```diff
@@ -322,6 +390,42 @@ struct GC
     }
 
 
+    private void runLocked(alias time, alias count)(scope void delegate() nothrow dg)
```

Comment:
I get up to a few percent slowdown (worst in tree1), apparently because the delegate call isn't inlined.